agent/pkg/controllers/config_controller.go

```diff
@@ -70,11 +70,54 @@ func PostTapConfig(c *gin.Context) {
 }
 
 func GetTapConfig(c *gin.Context) {
-	if globalTapConfig != nil {
-		c.JSON(http.StatusOK, globalTapConfig)
+	tappedNamespaces := make(map[string]bool)
+
+	if config.Config.IsNsRestrictedMode {
```

Comment:
i think we need to complicate our lives with this mode in standalone mode... just remove this support